app/helpers/email_helper.rb

```diff
@@ -2,7 +2,8 @@ module EmailHelper
   WHITEHALL_BOUND_BASE_PATH = "/world".freeze
 
   def render_whitehall_email_links?(base_path)
-    base_path.starts_with?(WHITEHALL_BOUND_BASE_PATH) if base_path
+    @presented_taxon.rendering_type == TaxonPresenter::ACCORDION &&
```

Comment:
I think this would be a bit easier to understand if the two parts of the condition were broken into methods e.g. `is_an_accordion? && handled_by_whitehall?(base_path)`